### DISCUSSION TITLE
想问一下EnableTimeoutCheck这个配置是起什么作用，配置文件没得配置，默认是打开的，但是我看备注是测试用

### DISCUSSION THREAD

**Discussion by zcl1115** [harmonycloud/kindling#606] [agent collector]
想问一下Network的EnableTimeoutCheck这个配置是起什么作用，配置文件没得配置，默认是打开的，但是我看备注是测试用

**Comment by hocktea214** [comment]
对于运行时，需定期清理失效、未匹配数据，EnableTimeoutCheck默认开启
而测试用例中不需要这些定时任务，通过该配置，关闭这些定时任务。
